ui/app/AppLayouts/Profile/Sections/HelpContainer.qml

```diff
@@ -7,33 +7,153 @@ import "../../../../shared/xss.js" as XSS
 
 Item {
     id: helpContainer
-    width: 200
-    height: 200
-    Layout.fillHeight: true
+    height: parent.height
     Layout.fillWidth: true
 
-    StyledText {
-        id: element8
-        //% "Help menus: FAQ, Glossary, etc."
-        text: qsTrId("help-menus:-faq,-glossary,-etc.")
-        anchors.left: parent.left
-        anchors.leftMargin: 24
-        anchors.top: parent.top
-        anchors.topMargin: 24
-        font.weight: Font.Bold
-        font.pixelSize: 20
-    }
 
-    // FIXME the link doesn't exist
-//    StyledText {
-//        anchors.centerIn: parent
-//        text: Utils.linkifyAndXSS(link)
-//        onLinkActivated: Qt.openUrlExternally(link)
-
-//        MouseArea {
-//            anchors.fill: parent
-//            acceptedButtons: Qt.NoButton // we don't want to eat clicks on the Text
-//            cursorShape: parent.hoveredLink ? Qt.PointingHandCursor : Qt.ArrowCursor
-//        }
-//    }
+    ScrollView {
+        height: parent.height
+        width: parent.width
+        contentHeight: glossary.height + linksSection.height + Style.current.bigPadding * 4
+        clip: true
+        Item {
+            id: glossary
+            anchors.top: parent.top
+            anchors.topMargin: 24
+            anchors.right: parent.right
+            anchors.rightMargin: contentMargin
+            anchors.left: parent.left
+            anchors.leftMargin: contentMargin
+            height: this.childrenRect.height
+
+            StyledText {
+                id: glossaryTitle
+                text: qsTr("Glossary")
+                anchors.left: parent.left
+                anchors.top: parent.top
+                font.pixelSize: 15
+                color: Style.current.secondaryText
+            }
+
+            GlossaryEntry {
+                id: entryAccount
+                anchors.top: glossaryTitle.bottom
+                anchors.topMargin: Style.current.padding
```

Comment:
FYI, it would have been easier to do this with a Column as the parent since that way you don't have to set the anchors.top for each Entry. It has a `spacing` property that handles equal spacing between each Entry.